.github/workflows/indent.yml

```diff
@@ -31,6 +31,7 @@ name: indent
 on:
   push: { branches: [ main ] }
   pull_request:
+    types: [ready_for_review]
```

Comment:
this is kind of unrelated, no?

Also , Draft PRs are a good way to get the tests to run before making the full PR IMHO